deploy/aws/cloudbeat-aws.yml

```diff
@@ -0,0 +1,66 @@
+cloudbeat:
+  type: cloudbeat/cis_aws
+  access_key_id: ""
+  secret_access_key: ""
+  # Defines how often an event is sent to the output
+  period: 30s
+  fetchers:
+    - name: aws-iam
+  evaluator:
+    decision_logs: false
+  runtime_cfg:
+    activated_rules:
+      cis_aws:
+        - cis_1_8
+        - cis_1_9
+# =================================== Kibana ===================================
+setup.kibana:
+
+  # Kibana Host
+  host: "http://localhost:5601"
+# =============================== Elastic Cloud ================================
+
+# These settings simplify using Cloudbeat with the Elastic Cloud (https://cloud.elastic.co/).
+
+# The cloud.id setting overwrites the `output.elasticsearch.hosts` and
+# `setup.kibana.host` options.
+# You can find the `cloud.id` in the Elastic Cloud web UI.
+#cloud.id:
+
+# The cloud.auth setting overwrites the `output.elasticsearch.username` and
+# `output.elasticsearch.password` settings. The format is `<user>:<pass>`.
+#cloud.auth:
+
+# ---------------------------- Elasticsearch Output ----------------------------
+output.elasticsearch:
+  # Array of hosts to connect to.
+  hosts: ["localhost:9200"]
+
+  # Protocol - either `http` (default) or `https`.
+#  protocol: "https"
+
+  # Authentication credentials - either API key or username/password.
+  #api_key: "id:api_key"
+#  username: ${ES_USERNAME}
+#  password: ${ES_PASSWORD}
+
+  # Enable to allow sending output to older ES versions
+  allow_older_versions: true
+
+# ================================= Processors =================================
+processors:
+  - add_host_metadata: ~
+  - add_cloud_metadata: ~
+  - add_docker_metadata: ~
+
```

Comment:
```suggestion
  - add_host_metadata: ~
  - add_cloud_metadata: ~
  - add_docker_metadata: ~
  - add_cluster_id: ~

```

Reply:
The `add_cluster_id` processor is not relevant for a non-k8s deployment, it will panic.